main.nf

```diff
@@ -0,0 +1,932 @@
+#!/usr/bin/env nextflow
+/*
+vim: syntax=groovy
+-*- mode: groovy;-*-
+
+========================================================================================
+               NGI - E X O S E Q    B E S T    P R A C T I C E
+========================================================================================
+ #### Homepage / Documentation
+ https://github.com/scilifelab/NGI-ExoSeq
+ #### Authors
+ Senthilkumar Panneerselvam @senthil10 <senthilkumar.panneerselvam@scilifelab.se>
+ Phil Ewels @ewels <phil.ewels@scilifelab.se>
+ Alex Peltzer @alex_peltzer <alexander.peltzer@qbic.uni-tuebingen.de>
+ Marie Gauder <marie.gauder@student.uni-tuebingen.de>
+----------------------------------------------------------------------------------------
+Developed based on GATK's best practise, takes set of FASTQ files and performs:
+ - alignment (BWA)
+ - recalibration (GATK)
+ - realignment (GATK)
+ - variant calling (GATK)
+ - variant evaluation (SnpEff)
+*/
+
+// Package version
+version = '0.8'
+
+// Help message
+helpMessage = """
+===============================================================================
+NGI-ExoSeq : Exome/Targeted sequence capture best practice analysis v${version}
+===============================================================================
+
+Usage: nextflow scilifelab/NGI-ExoSeq --reads '*_R{1,2}.fastq.gz' --genome GRCh37
```

Comment:
Capitalisation error - should be SciLifeLab

Reply:
Fixed in header 5499c18a3224ae292f2529fd8c8d70723809cae2